front-end/src/components/modal/create/GraphSettings.js

```diff
@@ -27,20 +34,21 @@ const GraphSettings = ({ movePage, setGraphType, setLiveCheck }) => {
         <div className="graphTypesBox">
           <div className="graphOptionsText">Graph Types</div>
           <div className="GraphTypeSelect">
-            <select id="graphTypeSelect" >
+            <select id="graphTypeSelect" onChange={(e) => { handleTypeSelect(e) }}>
               <option value="line">Line</option>
+              <option value="video">Video</option>
               <option value="spline">Spline</option>
               <option value="scatter">Scatter</option>
-              <option value="colour">XYColour</option>
+              <option value="coloredline">XYColour</option>
```

Comment:
Similar to what I said last time, lets move XY colour up since spline and scatter don't matter